File: 05_menuOperaciones/Csharp/menuOperSC.cs
using System;
public class menuOperSC
{
    public static void Main(String[] args)
    {
        double num1,num2,res; char op;
        Console.Write("Elije una opcion:\na) Suma.\nb) Resta.\nc) Multiplicacion.\nd) Division.\n Opcion: ");
        op = Convert.ToChar(Console.ReadLine());
        switch (op)
        {
            case 'a':
            {
                Console.Write("Escribe primer sumando: ");
                num1 = Convert.ToDouble(Console.ReadLine());
                Console.Write("Escribe el segundo sumando: ");
                num2 = Convert.ToDouble(Console.ReadLine());
                res=num1+num2;
                Console.WriteLine("La suma es: {0:G}",res);
                break;
            }
            case 'b':
            {
                Console.Write("Escribe el minuendo: ");
                num1 = Convert.ToDouble(Console.ReadLine());
                Console.Write("Escribe el sustraendo: ");
                num2 = Convert.ToDouble(Console.ReadLine());
                res=num1-num2;
                Console.WriteLine("La resta es: {0:G}",res);
                break;
            }
            case 'c':
            {
                Console.Write("Escribe primer factor: ");
                num1 = Convert.ToDouble(Console.ReadLine());
                Console.Write("Escribe el segundo factor: ");
                num2 = Convert.ToDouble(Console.ReadLine());
                res=num1*num2;
                Console.WriteLine("La multiplicacion es: {0:G}",res);
                break;
            }
            case 'd':
            {
                Console.Write("Escribe dividendo: ");
                num1 = Convert.ToDouble(Console.ReadLine());
                Console.Write("Escribe divisor: ");
                num2 = Convert.ToDouble(Console.ReadLine());
                res=num1/num2;
                Console.WriteLine("La division es: {0:G}",res);
                break;
            }
            default:
            {
                Console.WriteLine("Esa no es una de las opciones.");
                break;
            }
        }
    }
}

File: 05_menuOperaciones/C/menuOperSC.c
#include<stdio.h>
float num1,num2,res;
char op;
int main()
{
    printf("Elija una de las siguientes opciones:\n\n\ta) Suma.\n\tb) Resta.\n\tc) Multiplicacion.\n\td) Division.\n\n\t\t\t\t Opcion: ");
    scanf("%s",&op);
    switch(op)
    {
        case 'a':
        {
            printf("Escribe primer sumando: ");
            scanf("%f",&num1);
            printf("Escribe segundo sumando: ");
            scanf("%f",&num2);
            res=num1+num2;
            printf("La suma es: %f\n",res);
            break;
        }
        case 'b':
        {
            printf("Escribe el minuendo: ");
            scanf("%f",&num1);
            printf("Escribe el sustraendo: ");
            scanf("%f",&num2);
            res=num1-num2;
            printf("La resta es: %f\n",res);
            break;
        }
        case 'c':
        {
            printf("Escribe primer factor: ");
            scanf("%f",&num1);
            printf("Escribe segundo factor: ");
            scanf("%f",&num2);
            res=num1*num2;
            printf("El producto: %f\n",res);
            break;
        }
        case 'd':
        {
            printf("Escribe el dividendo: ");
            scanf("%f",&num1);
            printf("Escribe el divisor: ");
            scanf("%f",&num2);
            res=num1/num2;
            printf("La division es: %f\n",res);
            break;
        }
        default:
        {
            printf("Esa no es una opcion.\n");
        }
    }
    return 0;
}